src/functions.cpp

```diff
@@ -1,10 +1,10 @@
-#include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
+#include "functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
 #include <algorithm>
 #include <iostream>
 #include <limits>
 
 void print(std::vector<int> const & numbers) {
-    for (int i = 0; i < numbers.size(); ++i) {
+    for (long unsigned int i = 0; i < numbers.size(); ++i) {
```

Comment:
Using std::size_t will be a better option.

Reply:
Yup, please use size_t to get max points for this task.

Reply:
The compiler suggested using long unsigned int. Why should I use size_t? Is there any major difference between them?

Reply:
You can read here why size_t is better: 
https://en.cppreference.com/w/cpp/types/size_t